admin/class-clicky-admin-page.php

```diff
@@ -39,14 +39,14 @@ private function file_ext( $ext ) {
 	 * Enqueue the styles for the admin page.
 	 */
 	public function config_page_styles() {
-		wp_enqueue_style( 'clicky-admin-css', CLICKY_PLUGIN_DIR_URL . 'css/clicky_admin' . $this->file_ext( '.css' ), null, CLICKY_PLUGIN_VERSION );
+		wp_enqueue_style( 'clicky-admin-css', CLICKY_PLUGIN_DIR_URL . 'css/dist/clicky_admin.css', null, CLICKY_PLUGIN_VERSION );
```

Comment:
Maybe something should be added to the `CONTRIBUTING.md` file about the use of `grunt` ?

Reply:
yes, will do.